.github/workflows/_check-vars-and-secrets.yml

```diff
@@ -21,9 +26,32 @@ jobs:
     name: Check available vars and secrets
     runs-on: ubuntu-20.04
     steps:
+      - name: Debug input values
+        run: |
+          echo "${{ inputs.check-vars }} ${{ inputs.check-secrets }}"
+
+      - name: Check dev vars
+        run: |
+          if [[ "${{ inputs.check-dev-vars }}" != "true" ]]; then
+            echo "Skipping dev vars check"
+            exit 0
+          fi
+
+          if [[ \
+            -z "${{ vars.RUST_TOOLCHAIN_NIGHTLY }}" \
+          ]]; then
+            echo "!!! Some repository variables are either missing or empty."
+            echo "!!! Please check either repository or organization settings."
+            exit 1
+          fi
+
       - name: Check vars
-        if: inputs.check-vars == true
         run: |
+          if [[ "${{ inputs.check-vars }}" != "true" ]]; then
+            echo "Skipping vars check"
+            exit 0
+          fi
+
           if [[ \
             -z "${{ vars.ECR_PUBLIC_HOST }}" || \
             -z "${{ vars.ECR_PUBLIC_REGISTRY }}" || \
```

Comment:
Do we want to check `"${{ vars.RUST_TOOLCHAIN_NIGHTLY }}"` in both `dev-vars` and `vars`?

Reply:
Oh boy, yes, of course!  It's removed now.